src/main/java/com/wolox/training/controller/BookController.java

```diff
@@ -119,4 +120,21 @@ public void delete(@PathVariable(name = "id") int id) throws BookNotFoundExcepti
         bookRepository.deleteById(id);
     }
 
+    /**
+     * This method retrieves a List of {@link Book} following the next params.
+     *
+     * @param publisher: The publisher of Book for filter list
+     * @param genre:     The genre of Book for filter list
+     * @param year:      The year of Book for filter list
+     * @return List of {@link Book} filtered with the params passed
+     */
+    @GetMapping("{publisher}/{genre}/{year}")
```

Comment:
Those should be parameters, not path variables.

Reply:
I understand you, I will make the change

Reply:
Solved, I have changed the path parameters for the query parameters, the query parameters are mandatory because in the next card the issue to work on is that these parameters are optional